app/views/reports/regions/_hypertension_patient_coverage.html.erb

```diff
@@ -0,0 +1,84 @@
+<div class="d-flex ai-center jc-between">
+  <p class="mb-8px">
+    Hypertension patient coverage
+    <span class="p-relative t-1px">
+      <i
+        class="fas fa-question-circle fs-14px c-grey-dark"
+        data-toggle="tooltip"
+        data-placement="right"
+        data-trigger="hover click"
+        data-html="true"
+        title="<%= t('hypertension_patient_coverage_copy') %>"
+      >
+      </i>
+    </span>
+  </p>
+  <p class="mb-8px fw-bold">
+    <% if @region.estimated_population&.show_coverage %>
+      <span class="c-green-dark">
+        <%= number_to_percentage(@region.estimated_population.hypertension_patient_coverage_rate, precision: 0) %>
+      </span>
+    <% else %>
+      N/A
+    <% end %>
+  </p>
+</div>
+<div class="d-flex h-8px mb-16px o-hidden <% if @region.estimated_population&.hypertension_patient_coverage_rate == 100.0 %>br-4px<% end %>">
+  <div
+    class="br-tl-4px br-bl-4px <% if @region.estimated_population&.show_coverage %>bg-green-dark<% else %>bg-grey-light<% end %>"
+    style="width: <% if @region.estimated_population&.show_coverage %><%= number_to_percentage(@region.estimated_population.hypertension_patient_coverage_rate, precision: 0) %><% else %>5%<% end %>;"
+  ></div>
+  <div class="flex-1 br-tr-4px br-br-4px bg-grey-light"></div>
+</div>
+<div class="d-flex ai-center jc-between">
+  <p class="mb-8px">
+    Total registered patients
+    <span class="p-relative t-1px">
+      <i
+        class="fas fa-question-circle fs-14px c-grey-dark"
+        data-toggle="tooltip"
+        data-placement="right"
+        data-trigger="hover click"
+        data-html="true"
+        title="<%= t('registered_patients_copy.total_registered_patients', region_name: @region.name) %>"
+      >
+      </i>
+    </span>
+  </p>
+  <p class="mb-8px fw-bold">
+    <%= number_with_delimiter(@chart_data[:patient_breakdown][:total_patients]) %>
+  </p>
+</div>
+<div class="d-flex ai-center jc-between">
+  <p class="mb-8px">
+    Total est. hypertensive population
+    <span class="p-relative t-1px">
+      <i
+        class="fas fa-question-circle fs-14px c-grey-dark"
+        data-toggle="tooltip"
+        data-placement="right"
+        data-trigger="hover click"
+        data-html="true"
+        title="<%= total_estimated_hypertensive_population_copy(@region) %>"
+      >
+      </i>
+    </span>
+  </p>
+  <p class="mb-8px">
+    <% if @region.estimated_population&.show_coverage %>
+      <span class="fw-bold">
+        <%= number_with_delimiter(@region.estimated_population&.population) %>
+      </span>
+    <% elsif current_admin.power_user? || current_admin.accessible_facilities(:manage).any? %>
```

Comment:
This auth check doesn't seem quite right -- wouldn't we want to check something like `accessible_district?(@region, :manage)` for when they are looking at a district?  

For state reports, maybe just see if they have any accessible_districts at all w/i the state?

Reply:
Thanks Rob! `accessible_district?` doesn't exist, so I'm using `accessible_region?` instead. It's working on my local.